install/update_94_95.php

```diff
@@ -1004,7 +1004,7 @@ function update94to95() {
       'default_dashboard_central'     => 'central',
       'default_dashboard_assets'      => 'assets',
       'default_dashboard_helpdesk'    => 'helpdesk',
-      'default_dashboard_mini_ticket' => 'mini_ticket',
+      'default_dashboard_mini_ticket' => 'mini-tickets',
```

Comment:
Maybe add an update for this change

Reply:
did the inverse, re-set to the old value

Reply:
Finally, you just add a 's' to the key. But still if someone has already installed the current 9.5/bf version, you need to change `mini_ticket` to `mini_tickets` in its database, no?

Reply:
it shoult be better now